### DISCUSSION TITLE
Chromebook

### DISCUSSION THREAD

**Discussion by Texashuman** [Heroic-Games-Launcher/HeroicGamesLauncher#2369] [Q&A]
I have no idea what happened. Logged completely off the Ghromebook and back on and now the Heroic Launcher is in a LINUX APPS FOLDER and it ran and I was able to log into Gog and Epic. 

I just purchased an Acer Chromebook Vero 514 - CBV514-1H-55VP

I downloaded heroic_2.5.2_amd64.deb]
Followed the directions to load it.
I have an icon in the tray and when I click it to OPEN, it starts having a rotating circle thing like it is doing something, but even after an hour nothing has actually happened.

**Comment by jayjewel** [comment]
I have the same issue on my Galaxy Chromebook. I uninstalled everything and re-downloaded it, but now when I click the icon absolutely nothing happens, like it doesn't exist.

**Reply by Texashuman** [reply]
jayjewel. I have reach the conclusion that unless we can somehow get Google interested in the issues we are having, we ma not succeed.
OR,
Figure out a way to actually load a more complete version of Ubuntu Linux that GOG Games fully supports.

On 2.5.2 above I could get Master of Orion, the game I am most interested in, but also the old AD&D games to run and seem to play, but I couldn't get them into full screen. 
Not getting full screen seems to a problem with how GOG Games implemented DOSBOX.

I currently have Heroic working on my Acer using the FlatPak 2.7. distribution.

BUT.. Now I have an issue again with the GOG implementation of DOSBOX in that the graphics binary's are incomplete at least for Master of Orion II. I have not yet tried any of the old Gold Box AD&D games. Probably should though. HUmnnnnnnn.

I have been too busy with real life to dig into the Idea of getting a true Ubuntu Distro running and I am not a Linux aficionadoI, but I think it should be possible.
If so, then it will be possible to actually make use of the GOG Games Support system to get DOSBOX working properly since Gog is the people that did the Dosbox config for the Games they offer.
The DOSBOX people say that all they do is provide support for the original use of DosBox which was to run old Dos games on MS systems and even then, they expect the game player to edit the DosBox Config files to match the specific Games. They build in a LOT of controls and a pretty good explanation of how to use them.

**Reply by KonGames0** [reply]
so that means that the heroic launcher is broken on chrome os right?

**Reply by Texashuman** [reply]
Well,, It still isn't clear to me exactly what is broken.
The Heroic Launcher seems to work and I can see all my games.
So far I have tried only GOG games and I seem to always run into a problem with how they have implemented DOSBOX.
